### DISCUSSION TITLE
Make http headers for apollo explorer configurable at the programmatic level

### DISCUSSION THREAD

**Discussion by Axedyson** [apollographql/apollo-server#6724] [Ideas] [answered]
Would be nice if you can configure [http headers](https://www.apollographql.com/docs/studio/explorer/connecting-authenticating#request-headers) for apollo explorer in code instead of having to use the gui. This would eliminate an extra step for each developer having to set up these headers themselves

**Comment by glasser** [comment, marked as answer]
Do you mean when using the landing page? You can — install the plugin and pass the [`headers` option](https://www.apollographql.com/docs/apollo-server/api/plugin/landing-pages/#headers).

**Reply by Axedyson** [reply]
Oh so you can actually do that thanks!
